src/main.ts

```diff
@@ -2,18 +2,9 @@ import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
 import { SwaggerModule } from '@nestjs/swagger';
 import { swaggerConfig } from './config/swagger/swagger.config';
+import { initializeApp } from './config/bootstrap/app.initializer';
 
 async function bootstrap() {
-  const app = await NestFactory.create(AppModule);
-  // Set global prefix for all routes
-  app.setGlobalPrefix('api/v1');
-
-  // setting up swagger UI
-  const document = SwaggerModule.createDocument(app, swaggerConfig);
-  SwaggerModule.setup('api', app, document, {
-    jsonDocumentUrl: 'swagger/json',
-  });
-
-  await app.listen(3000);
+  initializeApp();
```

Comment:
I don't see the point of this file. Why don't you call `initializeApp` directly in the `config/bootstrap/app.initializer`?

Reply:
As in main.ts we need to show how and from where the app is being initialised.

Reply:
I think you should either implement `initializeApp` here and call that, or you can remove this file. 
I don't understand why do we need to create `bootstrap` function here. Plus, there are a lot of unused imports in this file.